tensorflow/examples/README.md

```diff
@@ -7,10 +7,10 @@ If that's not what you're looking for here are some links:
   [the tutorials on tensorflow.org](https://tensorflow.org/tutorials)
 * For community maintained keras examples goto [keras.io/examples](https://keras.io/examples/)
 * For TensorFlow Lite examples see [the tensorflow/examples repository](https://github.com/tensorflow/examples/tree/master/lite)
+* If you're looking for topics like course assignments, templates, then refer [the tensorflow/examples repository](https://github.com/tensorflow/examples/tree/master/lite)
```

Comment:
I'm not sure what "course assignments, templates" you're referring to here. The link goes to the lite directory.

Can you clarify? Maybe: "For the Udacity course notebooks see  [this directory](https://github.com/tensorflow/examples/tree/master/courses)"

Reply:
Hmm alright. I thought it was assignments 😅 coz the heading mentioned so! Anyway, updating this and yes, it redirects to lite directory

Reply:
I've updated it, please check 👍